src/api/spec/controllers/webui/users/tokens_controller_spec.rb

```diff
@@ -95,6 +95,18 @@
         it { expect { subject }.not_to change(Token, :count) }
       end
     end
+
+    context 'type is service' do
+      context 'and the description is larger than 64 characters' do
+        render_views
+
+        let(:form_parameters) { { token: { type: 'service', description: 'Something' } } }
+
+        include_examples 'check for flashing a success'
```

Comment:
You are testing `text_field`, I'm pretty sure the `ActionView` people already do that...